airbyte-integrations/connectors/source-snowflake/integration_tests/expected_spec.json

```diff
@@ -0,0 +1,232 @@
+                ],
+                "order": 15,
+                "title": "Schema",
+                "type": "string"
+            },
+            "streams": {
+                "description": "Used to filter out data on the source and creates a dedicated stream",
+                "items": {
+                    "properties": {
+                        "name": {
+                            "description": "The name that will be used to identify the pushdown filter stream in the connection (not nullable)",
+                            "title": "Name",
+                            "type": "string"
+                        },
+                        "namespace": {
+                            "description": "The namespace that will be used to identify the pushdown filter stream in the connection (nullable).",
+                            "title": "Namespace",
+                            "type": "string"
+                        },
+                        "parent_namespace": {
+                            "description": "The namespace of the relevant stream that the filter is being applied on (nullable).",
+                            "title": "Parent namespace",
+                            "type": "string"
+                        },
+                        "parent_stream": {
+                            "description": "The name of the relevant stream that the filter is being applied on (not nullable).",
+                            "title": "Parent stream",
+                            "type": "string"
+                        },
+                        "where_clause": {
+                            "description": "A string that must be a valid where clause in the relevant warehouse (not nullable).",
+                            "title": "Where Clause",
+                            "type": "string"
+                        }
+                    },
+                    "required": [
+                        "name",
+                        "parent_stream",
+                        "where_clause"
+                    ],
+                    "title": "FileBasedStreamConfig",
+                    "type": "object"
+                },
+                "order": 16,
+                "title": "Push Down Filter",
+                "type": "array"
+            },
+            "warehouse": {
+                "description": "The warehouse you created for Airbyte to access data.",
+                "examples": [
+                    "AIRBYTE_WAREHOUSE"
+                ],
+                "order": 13,
+                "title": "Warehouse",
+                "type": "string"
+            }
+        },
+        "required": [
+            "credentials",
+            "host",
+            "warehouse",
+            "database",
+            "role",
+            "replication_method"
+        ],
+        "title": "Snowflake Source Specification",
+        "type": "object"
+    },
+    "documentationUrl": "https:\/\/docs.airbyte.com\/integrations\/sources\/snowflake"
```

Comment:
Why are there `\` before `/`? It feels like those do not need to be escaped. The same comment applies to `$schema` but not to `replication_method.description`

Reply:
Done